gcloud-java-dns/src/main/java/com/google/gcloud/spi/DnsRpc.java

```diff
@@ -16,6 +16,12 @@
 
 package com.google.gcloud.spi;
 
+import com.google.api.services.dns.model.Change;
+import com.google.api.services.dns.model.ManagedZone;
+import com.google.api.services.dns.model.Project;
+import com.google.api.services.dns.model.ResourceRecordSet;
+import com.google.gcloud.dns.DnsException;
+
```

Comment:
Yes for maxResults. Not really for dnsName. The service allows two options: dnsName (e.g. for listing zones) and name (e.g. for listing dns records), so we need both options. I made a mistake of tossing both into dnsName. I am adding the option name option.